src/rez/solver.py

```diff
@@ -426,13 +428,17 @@ def key(variant):
                 if not request.conflict:
                     req = variant.requires_list.get(request.name)
                     if req is not None:
-                        requested_key.append((-i, req.range))
+                        orderer = get_orderer(req.name, self.solver.package_orderers or {})
```

Comment:
```suggestion
                        orderer = get_orderer(req.name, orderers=self.solver.package_orderers or {})
```
also, is `or {}` really necessary?

Reply:
I don't think so.  It was in there originally and tried hard not to change things that weren't absolutely necessary for this.